sections/k-partners-listing.liquid

```diff
@@ -0,0 +1,467 @@
+                                                        assign number_of_columns = 1
+                                                        assign number_of_columns_mobile = 1
+                                                        assign grid_space_desktop = number_of_columns | minus: 1 | times: settings.spacing_grid_horizontal | plus: 100 | append: 'px'
+                                                        assign grid_space_tablet = number_of_columns_mobile | minus: 1 | times: settings.spacing_grid_horizontal | plus: 100 | append: 'px'
+                                                        assign grid_space_mobile = number_of_columns_mobile | minus: 1 | times: settings.spacing_grid_horizontal | divided_by: 2 | plus: 30 | append: 'px'
+                                                        assign image_width = 1
+                                                    -%}
+                                                    {% capture sizes %}
+                                                        (min-width: {{ settings.page_width }}px) calc(({{ settings.page_width }}px - {{ grid_space_desktop }}) * {{ image_width }} /  {{ number_of_columns }}),
+                                                        (min-width: 990px) calc((100vw - {{ grid_space_desktop }}) * {{ image_width }} / {{ number_of_columns }}),
+                                                        (min-width: 750px) calc((100vw - {{ grid_space_tablet }}) * {{ image_width }} / {{ number_of_columns_mobile }}),
+                                                        calc((100vw - {{ grid_space_mobile }}) * {{ image_width }} / {{ number_of_columns_mobile }})
+                                                    {% endcapture %}
+                                                    {{
+                                                        object.logo
+                                                        | image_url: width: 300
+                                                        | image_tag:
+                                                        loading: 'lazy',
+                                                        widths: '50, 75, 100, 150, 200, 300, 400, 500, 750, 1000, 1250, 1500, 1750, 2000, 2250, 2500, 2750, 3000, 3200',
+                                                        sizes: sizes
+                                                    }}
+                                                </div>
+                                            {% else %}
+                                                {% render 'icon-setting' %}
+                                            {% endif %}
+                                        </div>
+                                        <div class="t-flex-column">
+                                            <a href="{{ link }}" target="_blank" class="link link-over accent-link"> {{ block.settings.link_label | replace: '[name]', name }} <span class="icon-tiny">{% render 'icon-tiny-arrow' %}</span></a>
```

Comment:
would prefer to use an icon indicating opening in new window, like this:
https://pictogrammers.com/library/mdi/icon/open-in-new/